### PR TITLE
Aura.View/helper 

### PR DESCRIPTION
**Pull request by jsurgeson**
Hi Paul, added a $pos param to the meta helper and added 2 methods to the scripts helper, inline script and conditional script, thought you might be interested.

Jeff Surgeson SA

### CODE REVIEW

**Comment by pmjones** [comment]
Hi jsurgeson -- I like the conditional bit, but what's the purpose of $pos in the others? It doesn't look like it's honored anywhere.

**Comment by pmjones** [comment]
Ah never mind, I see the asort() now.  So, the next question is, for metas, does the order matter?  (I can see where it might for scripts.)

**Comment by jsurgeson** [comment]
It probably does not, the intention was to be able to end up with:

meta charset="utf-8"
title
meta
meta

for standard nice html5 code layout, it solved getting the meta before the title, but not the title between the meta's, it was a while ago that i played with the aura view/layout. I need to redo a full view/layout aura domain again since you made the latest changes, started the other day again, but ran into my old multiple vhost issue again.

File: src/Helper/Metas.php [hunk 1]
```diff
@@ -85,8 +92,10 @@ public function addName($name, $content)
      */
     public function get()
     {
+    	asort($this->metas);
```
Comment (harikt) [inline src/Helper/Metas.php:95]: CS issue . Need to move a bit more . ie 4 spaces .

**Comment by harikt** [comment]
@pmjones as @jsurgeson mentioned for UTF-8 , it must be the first one . So order differs .

**Comment by pmjones** [comment]
OK, I see the usefulness with metas.  What I need you to do is modify the logic so that the metas are stored in this format:  $metas[$pos][] = $meta.  Then ksort($metas); foreach ($metas as $pos => $list) { foreach ($list as $meta) { output $meta; } }.  (That is the way the Aura\Signal\Manager does it.)

Regarding Scripts::addInline() ... it seems like inline scripts should probably be put actually inline, not captured for later output.

Regarding Scripts::addCond() ... I like that.

**Comment by pmjones** [comment]
I take it back on Metas and $pos; the way you did it is exactly the way Scripts does it.  I'll merge.

**Pull request closed without merging**